roles/confluent.schema_registry/tasks/main.yml

```diff
@@ -191,3 +191,7 @@
   when:
     - health_checks_enabled|bool
     - not ansible_check_mode
+
+- name: Register Cluster
+  include_tasks: register_cluster.yml
+  when: cluster_registry_cluster_name is defined
```

Comment:
can we name this variable something tied to the component like `schema_registry_cluster_registry_name`
open to suggestions

also same goes for the other components

Reply:
Happy to change to component_name_cluster_name if that suits?  So for example schema_registry_cluster_name

Reply:
ya! I think `component_name_cluster_name` works for me. We will want them to have the component name in there bc what if there is service colocation

oh also these variables need documentation, which may be kinda challenging. idk what happens if you have something like this in `confluent.variables/defaults/main.yml`:
```
### Cluster registry cluster name for Schema Registry cluster
schema_registry_cluster_name: 
```

If that will mean schema_registry_cluster_name is defined or not. gotta test.

Worst case you could have:
```
schema_registry_cluster_name: ""
```

and then have the conditional be:
```
when: schema_registry_cluster_name|length >0
```

But i like the defined option more